library/src/main/java/com/google/android/exoplayer/text/SubtitleParserHelper.java

```diff
@@ -179,8 +179,13 @@ private void handleSample(long sampleTimeUs, SampleHolder holder) {
       if (sampleHolder != holder) {
         // A flush has occurred since this holder was posted. Do nothing.
       } else {
-        this.result = new PlayableSubtitle(parsedSubtitle, subtitlesAreRelative, sampleTimeUs,
-            subtitleOffsetUs);
+        // it's more accurate to read startTime from eventTime compare to sampleTime
```

Comment:
Oh right, I see what this is for. Although maybe we could clean this up a bit by doing the following:

- If there aren't any events in the subtitle, this.result may as well just be set to null, I think?
- There's no need for the PlayableSubtitle constructor to take startTimeUs. It should be able to calculate it internally.